src/observable/ConnectableObservable.ts

```diff
@@ -5,10 +5,10 @@ import {Subscription} from '../Subscription';
 
 export class ConnectableObservable<T> extends Observable<T> {
 
-  subject: Subject<T>;
-  subscription: Subscription;
+  protected _subject: Subject<T>;
```

Comment:
Is there reason to make private field starts with `_`? Current codebase usually do not use those convention only except if field has separate getter / setter.